tests/integration/kubernetes/gha-run.sh

```diff
@@ -151,6 +151,11 @@ function deploy_kata() {
 }
 
 function run_tests() {
+    platform="${1:-}"
+
+    [ "$platform" = "kcli" ] && \
+        export KUBECONFIG="$HOME/.kcli/clusters/${CLUSTER_NAME:-kata-k8s}/auth/kubeconfig"
```

Comment:
I would define a default `KCLI_CLUSTER_NAME` at the top level so that we don't repeat this `kata-k8s` string all over. Or even put this export in a function.